xml/security_ldap.xml

```diff
@@ -398,266 +399,195 @@ objectclass (2.16.840.1.113730.3.2.333 NAME 'nsPerson' <co xml:id="co-ldap-schem
   </variablelist>
 
   <sect2 xml:id="sec-security-ldap-server-instance" >
-  <title>Creating the &ds389; Instance</title>
+  <title>Creating a &ds389; Instance with an .inf File</title>
     <para>
-     You create the instance with the <command>dscreate</command> command.
-     It can take a configuration file (<filename>*.inf</filename>) which
-     defines the instance configuration settings. Alternatively, the command can
-     be run interactively.
+     Use the <command>dscreate</command> command to create a new &ds389; instance.
+     There are several ways to create a new instance: interactively, from a 
+     template file, using &yast;, and using a custom configuration file.
+     In this section you will learn how to use a custom configuration file. This 
+     file must have an <filename>.inf</filename> extension. 
+    </para>
+    <para>
+       The default instance name is <literal>localhost</literal>. The instance 
+       name cannot be changed after it has been created. It is better
+       to create your own instance name, rather than using the default, to avoid 
+       confusion and to enable a better understanding of how it all works.
     </para>
-    <note>
-     <title>Instance Name</title>
-      <para>
-       If not specified otherwise, the default instance name is
-       <literal>localhost</literal>. The instance name cannot
-       be changed after the instance has been created.</para>
-     <para>
-      To check the name of an instance you have not created yourself, use
-      the <command>dsctl -l</command> command.</para>
-    </note>
     <para>
      <xref linkend="ex-ldap-389-ds-inf" xrefstyle="select:label"/> shows an
-     example configuration file that you can use as a starting point. Alternatively,
-     use <command>dscreate create-template</command> to create a template
-     <filename>*.inf</filename> file. The template is commented and pre-filled,
-     so you can adjust its variables to your needs. For more details, see the
-     man page of <command>dscreate</command>.
+     example configuration file that you can use to create a new &ds389; instance.
+     You may copy and use this file, though be sure to create your own password.
     </para>
    <procedure>
     <step>
      <para>
-      If you want to set up a trial instance, start an editor and save the
-      following as <filename>/tmp/instance.inf</filename>:
+      Copy the following example file, <filename>389ds-test.inf</filename>, to 
```

Comment:
We made a change for SLE 15 SP3 to make the "example" generated much cleaner and direct, so we could consider just using the example generation command here? I can't remember if we backported it to SP2 though .... so not a "we must change this" but we "could" change it.

Reply:
If you mean `dscreate interactive`, that is in SP2, and I will create an additional section for it.

Reply:
Nah, the `dscreate create-template` command can generate the example config rather than asking people to copy it from the docs :)